scripts/travis/upload-to-docker.sh

```diff
@@ -18,7 +18,9 @@ else
   # TODO why do we do /// ?
   TAG="${BRANCH///}"
 fi
-echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, REPO=$REPO, BRANCH=$BRANCH, TAG=$TAG, IMAGE=$IMAGE"
+echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, REPO=$REPO, BRANCH=$BRANCH, TAG=$TAG, IMAGE=$IMAGE, SHA=$TRAVIS_COMMIT"
+
+docker tag $IMAGE $REPO:$TRAVIS_COMMIT
```

Comment:
Is that enough? Shouldn't you do a `docker push $REPO:$TRAVIS_COMMIT` as well?

Reply:
It should be enough. Script creates multiple tags and pushes them all with `push $REPO`